tests/testthat/test-enw_get_cache.R

```diff
@@ -0,0 +1,32 @@
+test_that("enw_get_cache can retrieve the enw_cache_location as expected", {
+    withr::with_envvar(new = c(enw_cache_location = "models"), 
+    expect_equal(
+        basename(enw_get_cache()), 
+        "models"
+    )
+    )
+
+    withr::with_envvar( new = c(enw_cache_location = ""),
+    expect_equal(
```

Comment:
is being inline with withr:: common practise? It looks a little weird to me?

Reply:
I added at the top of the library calls for `testthat`. To your point, it looks much more clean...